install-dependencies.sh

```diff
@@ -152,7 +152,7 @@ fedora_python3_packages=(
 # an associative array from packages to constrains
 declare -A pip_packages=(
     [scylla-driver]=
-    [geomet]="<0.3,>=0.1"
+    [geomet]=
```

Comment:
When checking locally, i noticed that indeed the scylla-driver release is 3.29.0 (which is not good), only after removing the `geomet` constrain i manage to get `scylla-driver=3.29.3`

@avikivity @mykaul Can this be removed?

Reply:
I have no idea what it is used for... I've tracked it all the way back to https://github.com/scylladb/scylladb/commit/159bc7c7ea031f05715d7c402aa038781f773264 - so perhaps @syuu1228 would know.

Reply:
If we remove it, it has to be on master first.

Reply:
Ok, can we remove it with the scylla-driver restriction (in master)?

Reply:
The scylla-driver restriction has to match tools/cqlsh/requirements.txt

Reply:
Good idea, but via master.